src/List/__docs__/item.md

```diff
@@ -0,0 +1,17 @@
+# Item
+
+The `Item` component is a direct child of the `Items` fragment.
+
+## Usage
+
+See `List`.
+
+## Props
+
+Prop Name | Required? | Description
+--------- | :-------: | :----------
+`classes` | ❌ | A classname hash
```

Comment:
Would be clearer to say `object` instead of `hash`. Also worth noting here what the keys/values should be for those not familiar with our css module setup already (they just want to try a component asap).